app/dao/notifications_dao.py

```diff
@@ -541,14 +543,28 @@ def dao_set_created_live_letter_api_notifications_to_pending():
     this is used in the run_scheduled_jobs task, so we put a FOR UPDATE lock on the job table for the duration of
     the transaction so that if the task is run more than once concurrently, one task will block the other select
     from completing until it commits.
+
+    Note - do not process services that have letters_as_pdf permission as they
+           will get processed when the letters PDF zip task is created
     """
+
+    # Ignore services that have letters_as_pdf permission
+    services_without_letters_as_pdf = [
+        s.id for s in Service.query.filter(
+            ~Service.permissions.any(
+                ServicePermission.permission == 'letters_as_pdf'
+            )
+        ).all()
+    ]
+
     notifications = db.session.query(
         Notification
     ).filter(
         Notification.notification_type == LETTER_TYPE,
         Notification.status == NOTIFICATION_CREATED,
         Notification.key_type == KEY_TYPE_NORMAL,
-        Notification.api_key != None  # noqa
+        Notification.api_key != None,  # noqa
+        Notification.service_id.in_(services_without_letters_as_pdf)
```

Comment:
Might be worth checking the performance of this. You may want to get the list of service ids outside of this query rather than a subquery.

Reply:
The join appears to be marginally faster than the sub query, so will implement that.